### ISSUE TITLE
tabs in yml files

### ISSUE THREAD

**Issue by sunnysideup** [mateusz/silverstripe-testdata#9]
Hi

Congrats on this great module... THANK YOU..... LIFE SAFER

I believe tabs in yaml files are illegal, but you list them in your example.

Nicolaas

**Comment by mateusz** [comment]
Yeah, they are illegal, but I think SilverStripe Yaml parser accepts them. Good idea to rewrite to conform with the standard - this module is due a good rewrite anyway, because the upstream Yaml API is quite volatile - it has been completely rewritten in 3.0. Will leave that open for now, so I can keep in mind.
